sync/clients/databricks.py

```diff
@@ -94,12 +143,22 @@ def _send(self, request: httpx.Request) -> dict:
         return {"error_code": "UNKNOWN_ERROR", "message": "Transaction failure"}
 
 
-_sync_client: DatabricksClient = None
+_databricks_config: DatabricksConf | None = None
+
+
+def get_databricks_config() -> DatabricksConf:
+    global _databricks_config
+    if not _databricks_config:
+        _databricks_config = DatabricksConf()
+    return _databricks_config
```

Comment:
I split this out so that I could query it independently here - https://github.com/synccomputingcode/syncsparkpy/pull/12/files#diff-fcb826fa7c84469280d545c2fb63c0ad6e4d83d38a3cff66ac7cd3d4c6b0ea5eR150

Reply:
I'm not sure this really belongs here. I've been doing,
```
from sync.config import CONFIG
```
which is provided by: https://github.com/synccomputingcode/syncsparkpy/blob/2919c60058acd73889f767d97593ec87dec62d69/sync/config.py#L148-L172

Reply:
ah, duh - so I should be able to just `from sync.config import DB_CONFIG`?

Reply:
Hmm - this does bring up the question of how to easily alter this global constant for tests. I guess some sort of monkeypatching?

Reply:
Yeah, I was thinking something similar to what we do in sync_backend, e.g. https://github.com/synccomputingcode/sync_backend/blob/ca26316173212ec225cefef803c463d00cc5d721/tests/unit/spark_predictor/web_tools/test_aws_ebs_pricing.py#L58-L62